frontend/src/components/atoms/SnackBarContainer.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from 'styled-components';
+import { theme } from '../../styling/theme';
+import SnackBarCloseButton from './SnackBarCloseButton';
+import SnackBarLoader from './SnackBarLoader';
+
+interface ISnackBarProps {
+  className?: string;
+  clicker?: () => void;
```

Comment:
Name should be updated, this doesn't make it very clear what it does. If it's a handler, use the appropriate type.